CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## [2.0.2]
+
+- **BREAKING CHANGE**:
```

Comment:
As this is a breaking change,  it should be version 3.0.0, check [pub versioning philosophy](https://dart.dev/tools/pub/versioning).